src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -8,17 +8,19 @@ import {
   FormGroup,
   Input,
 } from 'reactstrap';
-import fetchCielConcepts, { addExistingBulkConcepts, isConceptValid } from '../../redux/actions/bulkConcepts';
+import fetchSourceConcepts, { addExistingBulkConcepts, isConceptValid, getConceptSources } from '../../redux/actions/bulkConcepts';
```

Comment:
Shouldn't the above be named fetchConceptSources

Reply:
fixed